index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="header">
+        <div class="header1">
+            Naofumi&nbsp;Seki
+        </div> 
+        
+    
+    </div>
+
+    <div class="main">
+        
+    <img src="images/S__5701640.jpg" class="enoshima">
+    <div class="profile"> 
+         <p>
+        Profile
+        <li>2001/10/24</li>
+        <li>政治経済学部</li>
+        <Li>神奈川県</Li>
+        <li>湘南高校</li>
+        </p>
```

Comment:
基本的には見出しはh2~h?を使ってあげます！liはulで囲うのが基本的な使い方です！
```suggestion
        <h2>Profile</h2>
        <ul>
        <li>2001/10/24</li>
        <li>政治経済学部</li>
        <li>神奈川県</li>
        <li>湘南高校</li>
        </ul>
```